.devcontainer/ubuntu-gui-full/devcontainer.json

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2024 Antmicro <www.antmicro.com>
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  "image": "ghcr.io/antmicro/openroad-flow-scripts:lxqt",
```

Comment:
I would rather see PII responsible for the distribution of OR.  @vvbandeira to suggest the process.